client/blocks/upwork-stats-nudge/style.scss

```diff
@@ -0,0 +1,78 @@
+.card.upwork-stats-nudge {
+	padding: 0; // allow section header to perfectly sit over the top of the card
+}
+
+.upwork-stats-nudge__body {
+	align-items: flex-end;
+	display: flex;
+	flex-direction: row;
+	padding: 11px 24px 11px 24px; // Standard padding
+
+	@include breakpoint( '>960px' ) {
+		max-height: 300px;
+		padding-bottom: 0px; // Allow image to go right to the edge of the card
+	}
+}
+
+.upwork-stats-nudge__info {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+
+	@include breakpoint( '>960px' ) {
+		padding-right: 12px; // The image has about 12 px of whitespace, this helps "balance" the block
+	}
+}
+
+.upwork-stats-nudge__title {
+	font-size: 21px;
+	font-weight: 400;
+	margin: 10px 0 5px;
+}
+
+.upwork-stats-nudge__description {
+	font-size: 16px;
+	font-weight: 400;
+}
+
+.upwork-stats-nudge__button-row {
+	align-content: center;
+	display: flex;
+	flex-direction: row;
+	margin-bottom: 20px;
+
+	@include breakpoint( '>960px' ) {
+		margin-bottom: 30px;
+	}
+
+	.button {
+		@include breakpoint( '<480px' ) {
+			width: 100%;
+			text-align: center;
+		}
+	}
+}
+
+.upwork-stats-nudge__image-wrapper {
+	display: none;
+
+	@include breakpoint( '>960px' ) {
+		display: flex;
+		height: 200px;
+		margin-right: 40px;
+		min-width: 260px;
+	}
+}
+
+.upwork-stats-nudge__image {
+	width: 100%;
+}
+
+.upwork-stats-nudge__close-icon {
+	position: absolute;
+	top: 14px;
+	right: 18px;
+	color: $gray;
+	cursor: pointer;
+	z-index: 1;
```

Comment:
Shouldn't we use the [`z-index()` SASS function](https://github.com/Automattic/wp-calypso/blob/a68287dcffe461a69f1b91370576130ea785c726/assets/stylesheets/shared/functions/_z-index.scss) here?

Reply:
fixed!